packages/components/src/composite/types.ts

```diff
@@ -6,13 +6,13 @@ import type * as Ariakit from '@ariakit/react';
 export type CompositeContextProps =
```

Comment:
TODO: consider changing the context type to something like

```ts
export type CompositeContextProps = {
	/**
	 * The component store, used for advanced usage of the component.
	 */
	store?: Ariakit.CompositeStore;
};
```

for easier prop destructuring when consuming it